Makefile

```diff
@@ -43,13 +43,13 @@ export CURRENT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD;)
 export IMAGE_TAG?=127.0.0.1:5000/openshift/origin-$(OPERATOR_NAME):$(CURRENT_BRANCH)
 BUNDLE_TAG=$(error set OVERLAY to deploy or run a bundle)
 
-export LOGGING_VERSION?=5.8
+export LOGGING_VERSION?=5.9
```

Comment:
`OPENSHIFT_VERSIONS` in this file needs a bump to `v4.13-v4.17`. Looks fine otherwise :+1: